lib/main.coffee

```diff
@@ -11,6 +11,8 @@ Config = require './config'
 KernelManager = require './kernel-manager'
 Inspector = require './inspector'
 AutocompleteProvider = require './autocomplete-provider'
+zmq_services = require './zmq-services'
```

Comment:
@rgbkrk Could you comment about the globals defined by `jupyter-js-services`?
Does this interfere with any of the globals that Atom itself overwrites?
Is `nteract` already using `jupyter-js-services` as dependency?